Sources/Sentry/SentryTracer.m

```diff
@@ -320,10 +349,29 @@ - (void)addMeasurements:(SentryTransaction *)transaction
         }
 
         if (type != nil) {
-            [transaction setMeasurementValue:@{ @"value" : @(appStartMeasurement.duration * 1000) }
+            [transaction setMeasurementValue:@{ valueKey : @(appStartMeasurement.duration * 1000) }
                                       forKey:type];
         }
     }
+
+#if SENTRY_HAS_UIKIT
+    // Frames
+    SentryFramesTracker *framesTracker = [SentryFramesTracker sharedInstance];
+    if (framesTracker.isRunning && !_startTimeChanged) {
+        NSInteger totalFrames = framesTracker.currentTotalFrames - initTotalFrames;
+        NSInteger slowFrames = framesTracker.currentSlowFrames - initSlowFrames;
+        NSInteger frozenFrames = framesTracker.currentFrozenFrames - initFrozenFrames;
```

Comment:
M: should we validate that those numbers make sense? eg none of them should be <= 0, we do on Android